modules/microshift-exposed-audit-ports-hostport.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-exposed-audit-ports-hostport_{context}"]
+== hostport
+
+The following logs are examples for the hostport setting
+
+//what is the command to access these logs
```

Comment:
Was wondering if you knew if there was a command for accessing the logs?

Reply:
`journalctl -u crio | grep "local port"`